software/trailbase.yml

```
@@ -0,0 +1,13 @@
+name: TrailBase
+website_url: https://trailbase.io/
+description: Open, sub-millisecond, single-executable FireBase alternative with type-safe REST & realtime APIs, built-in JS/TS runtime, auth & admin UI built on Rust, SQLite & V8.
+licenses:
+  - OSL-3.0
+platforms:
+  - Rust
+  - Docker
+  - K8S
```

```suggestion
platforms:
  - Rust
  - Docker
```

I couldn't find the documentation for Kubernetes

---

https://github.com/trailbaseio/trailbase/tree/main/deploy/k8s only stuff I could find is here. There's no documented setup, but there are files for people to use.

---

Happy either way. Removed it for now. Not sure, how the instructions were to interpret:

> Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.

Neither docker nor k8s are strict requirements but it runs perfectly fine with either. We provide setups for both. I don't think it makes sense for TrailBase (or any similar project) to explain k8s in great detail that would be a bottomless pit.